src/PrestaShopBundle/ApiPlatform/Resources/Language.php

```diff
@@ -44,7 +44,7 @@
                 '[date_format_full]' => '[dateTimeFormat]',
                 '[is_rtl]' => '[isRtl]',
             ],
-            queryBuilder: 'prestashop.core.grid.query.builder.language',
+            gridDataFactory: 'prestashop.core.grid.factory.language_decorator',
```

Comment:
Maybe we can add the `flag` field in the Language API now that the factory returns this extra data? It would validate the new behavior